# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/uClibc-0.9.31.1-fix-error-locale-utf-8.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

---
 extra/locale/gen_wc8bit.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: uClibc-0.9.31/extra/locale/gen_wc8bit.c
===================================================================
--- uClibc-0.9.31.orig/extra/locale/gen_wc8bit.c
+++ uClibc-0.9.31/extra/locale/gen_wc8bit.c
@@ -120,7 +120,7 @@
 		}
 
  locale_failure:
-		printf("could not find a UTF8 locale ... please enable en_US.UTF-8\n");
+		fprintf(stderr, "could not find a UTF8 locale ... please enable en_US.UTF-8\n");
 		return EXIT_FAILURE;
  locale_success:
 		pclose(fp);
